static/js/apps/eval_embeddings/app.tsx

```diff
@@ -16,84 +16,89 @@
 
 import axios from "axios";
 import * as CSV from "csv-string";
+import _ from "lodash";
 import React, { useEffect, useRef, useState } from "react";
 
-import { loadSpinner, removeSpinner } from "../../shared/util";
-import { QuerySection } from "./query_section";
+import { MemoIndexScoreBox } from "./index_score_box";
 import { EmbeddingObject } from "./util";
 
-const DESCRIPTION_TEXTAREA_ID = "description-textarea";
-
-const DEFAULT_DESCRIPTION =
-  'Count_Person,"population number"\ndc/topic/AgeMedians,"median age related"';
-const DEFAULT_QUERIES = "how many population\nmedian age";
+const DEFAULT_DESCRIPTION = 'Count_Person,"population number"';
+const DEFAULT_QUERY_STRING = "how many population";
+const DEFAULT_INDEX = "base_uae_mem";
```

Comment:
any way to highlight that this is prod by perhaps showing this as "base_uae_mem (prod)".  And perhaps as "medium_ft (custom-dc)"?

Reply:
updated